packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -102,7 +102,7 @@ class MenuItem extends React.Component {
     return (
       <li
         className={cx('list-item')}
-        aria-selected="false"
+        aria-selected={isSelected}
```

Comment:
does this get passed as a string value?

Reply:
React converts it to a string either way, but it's a bool.